src/stylesheets/elements/_inputs.scss

```diff
@@ -77,8 +77,19 @@ select {
   label {
     margin-top: 0;
   }
+
+  .usa-input-inline {
+    border: $input-border-width solid $color-gray;
+    width: inherit;
+  }
+
+  .usa-input-inline-error {
+    border: 3px solid $color-secondary-dark;
```

Comment:
💭  might look a bit cleaner by having one or the other. Either way, when testing out the proposed implementation I get the following layout without that width value added in:

![screen shot 2016-06-15 at 3 49 47 pm](https://cloud.githubusercontent.com/assets/955558/16095020/d01fb426-3310-11e6-98fd-b67f0b72cbf5.png)